v2/pkg/engine/resolve/resolve_test.go

```diff
@@ -2213,9 +2213,9 @@ func TestResolver_ResolveGraphQLResponse(t *testing.T) {
 					},
 				},
 			},
-		}, Context{ctx: context.Background()}, `{"errors":[{"message":"Array cannot represent non-array value.","path":[]}],"data":null}`
```

Comment:
These are internal errors that were exposed because we didn't validate the response.